docs/structure-of-a-contract.rst

```diff
@@ -129,7 +129,7 @@ Structs are custom defined types that can group several variables (see
 Enum Types
 ==========
 
-Enums can be used to create custom types with a finite set of values (see
+Enums can be used to create custom types with a finite set of 'constant values' (see
```

Comment:
Perhaps even more refined: `create custom types with values from a finite set of constants`?

Reply:
A data member of Class/Struct/Enum always hold '**value**'. It's either '**variable**' or '**constant**'. So, the **nature** of the data member is 'variable' or 'constant'. So, it would still be **better** to say '**constant values**'.
Nevertheless, the 'original phrase' is still sounds good. And replacing it with either 'constants' or 'constant values' should be fine with me.

**Original:**
Enums can be used to create custom types with a finite set of 'values'.

**To be changed to EITHER:**
1.
Enums can be used to create custom types with a finite set of '*constant values*'
**OR:**
2.
Enums can be used to create custom types with a finite set of '*constants*'.

**NOTE:**
My preference is 1.

Reply:
Thanks, for the approval 👍